src/store/lookup.ts

```diff
@@ -145,6 +145,35 @@ async function lookupCardInStock(store: Store, page: Page, link: Link) {
 		}
 	}
 
+	if (store.labels.maxPrice && Config.store.maxPrice) {
```

Comment:
Do you mind abstracting this logic in this if block just like the if statements above and below using the `pageIncludesLabels`?

It would help with complexity.

That being said, take a look at that function. I believe what you're wanting is something very similar. As I think your approach is great, but I think there is some extra work you're doing here that may not be needed since we have those helper functions.

Reply:
Yes no worries, I'll take a look, appreciate the feedback! :sunglasses:

Reply:
Thanks! Look forward to see what's to come.

Reply:
Abstraction injection sequence complete 😄 
- Could not utilize `pageIncludesLabels` directly - as a mathematical comparison was needed, but it became a good jumping off point as reference for a new helper function...  
 Open to feedback 👍